WiserTaskScheduler/WiserTaskScheduler/Modules/Branches/Services/BranchBatchLoggerService.cs

```diff
@@ -0,0 +1,207 @@
+                                        INSERT INTO {WiserTableNames.WiserBranchMergeLog} 
+                                        (
+                                            branch_queue_id,
+                                            branch_queue_name,
+                                            branch_id,
+                                            date_time,
+                                            history_id,
+                                            table_name,
+                                            field,
+                                            action,
+                                            old_value,
+                                            new_value,
+                                            object_id_original,
+                                            object_id_mapped,   
+                                            item_id_original,
+                                            item_id_mapped,
+                                            item_entity_type,
+                                            item_table_name,
+                                            link_id_original,
+                                            link_id_mapped,
+                                            link_destination_item_id_original,
+                                            link_destination_item_id_mapped,
+                                            link_destination_item_entity_type,
+                                            link_destination_item_table_name,
+                                            link_type,
+                                            link_ordering,
+                                            link_table_name,
+                                            item_detail_id_original,
+                                            item_detail_id_mapped,
+                                            item_detail_language_code,
+                                            item_detail_group_name,
+                                            file_id_original,
+                                            file_id_mapped,
+                                            used_merge_settings,
+                                            used_conflict_settings,
+                                            production_host,
+                                            production_database,
+                                            branch_host,
+                                            branch_database,
+                                            status,
+                                            message
+                                        )
+                                        VALUES {String.Join($",{Environment.NewLine}\t", values)}
+                                        """;
+
+            await mySqlCommand.ExecuteNonQueryAsync();
+        }
+        catch (Exception exception)
+        {
+            logger.LogError(exception, "Error during log flush");
```

Comment:
Is there a specific reason the LogService of the WTS is not used?

Reply:
I forgot it exists.